packages/web-components/tests/snapshots/dds-text-cta.md

```diff
@@ -24,7 +24,7 @@
 
 ```
 <a
-  class="bx--link bx--link-with-icon bx--link-with-icon__icon-right"
+  class="bx--link bx--link--lg bx--link-with-icon bx--link-with-icon__icon-right"
```

Comment:
Is there a reason we need to touch so many components to make this change? Are we not depending on link with icon in these higher up components?

Reply:
We are depending on link with icon. These are the snapshot files that automatically get updated in web components. The only thing I touched was adding the large size in Link with Icon, which is now being showed everywhere link with icon is used.

Reply:
I see now. I saw snapshot in a couple of the file names, but missed it in some of the others it appears. Thanks for clarification!